integration_tests/seeds/hubspot_engagement.csv

```diff
@@ -0,0 +1,11 @@
+id,_fivetran_synced,active,activity_type,created_at,last_updated,owner_id,portal_id,timestamp,type
```

Comment:
I think we will need to remove UTC strings on the `_fivetran_synced` fields for Snowflake Buildkite errors to be fixed.

Reply:
This fixed it!